### PR TITLE
Adds "redliningchanged"-event to redlingin container.

### PR DESCRIPTION
**Pull request by KaiVolland**
This adds an event that fires everytime a feature is added, deleted, moved or modified.

### CODE REVIEW

File: src/view/container/Redlining.js [hunk 1]
```diff
@@ -222,6 +222,7 @@ Ext.define("BasiGX.view.container.Redlining", {
 
        if (!me.redliningVectorLayer) {
            me.redlineFeatures = new ol.Collection();
+           me.redlineFeatures.on('propertychange', me.fireRedliningChanged, me);
```
Comment (marcjansen) [inline src/view/container/Redlining.js]: When do you unregister this eventlistener?

File: src/view/container/Redlining.js [hunk 2]
```diff
@@ -637,6 +646,9 @@ Ext.define("BasiGX.view.container.Redlining", {
      */
     setState: function(state) {
         var me = this;
+
+        me.stateChangeActive = true;
```
Comment (buehner) [inline src/view/container/Redlining.js]: should be declared as member?

File: src/view/container/Redlining.js [hunk 3]
```diff
@@ -668,5 +680,12 @@ Ext.define("BasiGX.view.container.Redlining", {
         // reapply the styleFn on the layer so that ol3 starts redrawing
         // with new styles
         me.redliningVectorLayer.setStyle(me.redliningVectorLayer.getStyle());
+
+        me.stateChangeActive = false;
```
Comment (buehner) [inline src/view/container/Redlining.js]: see above

**Comment by buehner** [comment]
just minor comments, else good to go :+1:

File: src/view/container/Redlining.js [hunk 4]
```diff
@@ -668,5 +680,12 @@ Ext.define("BasiGX.view.container.Redlining", {
...
+    },
+
+    fireRedliningChanged: function(){
+        this.fireEvent('redliningchanged', this, this.getState(),
```
Comment (marcjansen) [inline src/view/container/Redlining.js]: Please add very basic API docs:

```js
/**
 * @event redliningchanged
 * Fires when …
* @param {type} variablename One sentence description. 
 */
```

to the class definition (see [here for details](https://github.com/senchalabs/jsduck/wiki/@event)).

**Comment by marcjansen** [comment]
Very nice, two minor remarks, otherwise good to go. Thanks @KaiVolland.

**Comment by KaiVolland** [comment]
I addressed your remarks.

**Pull request merged**